### PR TITLE
[Generic] RSS adding item description

### PR DESCRIPTION
**Pull request by renalid**
Add description of RSS feed's items

## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information
Adding the RSS feed items description if it's available in order to get complete information from the feed.

### CODE REVIEW

**Review by dstftw — CHANGES_REQUESTED** [review]
Add a test.

**Comment by renalid** [comment]
@dstftw test added. Thanks :-)

File: youtube_dl/extractor/generic.py [hunk 1]
```diff
@@ -2194,11 +2212,15 @@ def _extract_rss(self, url, video_id, doc):
 
             if not next_url:
                 continue
-
+            
+            item_desc_el = it.find('description')    
+            item_desc = None if item_desc_el is None else item_desc_el.text
```
Comment (dstftw) [inline youtube_dl/extractor/generic.py:2217]: Use `xpath_text` instead.

File: youtube_dl/extractor/generic.py [hunk 2]
```diff
@@ -214,6 +214,24 @@ class GenericIE(InfoExtractor):
...
+            'playlist': [{
+                'info_dict': {
+                    'id': '14631-22.11.2020-ITEMA_22491517-2020Y24252S0327',
+                    'ext': 'mp3',
+                    'title': 'Les musiques de Valérie Lemercier',
+                    'description': 're:.*occasion de la sortie.*',
+                    'upload_date': '20201122',
+                },
+            }],
+        },
```
Comment (dstftw) [inline youtube_dl/extractor/generic.py:234]: Just add description check to http://podcastfeeds.nbcnews.com/audio/podcast/MSNBC-MADDOW-NETCAST-M4V.xml test instead.

**Review by dstftw — CHANGES_REQUESTED** [review]

**Pull request merged**